pandas/guide/_sources/pandas_pr.rst.txt

```diff
@@ -69,22 +69,40 @@ Then, follow the next steps:
 where `<modified-file(s)>` is the file where you made your changes (in rare
 cases it could be more than one file). And `<commit-message>` is a short
 description of your changes, starting by "DOC:" (e.g. "DOC: Improved the
-docsting of DataFrame.head()").
+docstring of DataFrame.head()").
 
-5. Push your changes to the sprint repository
+5. Push your changes to the pandas repository
 ---------------------------------------------
 
-Before sending the pull request to the pandas core developers, you will send
-it to the sprint repository. Where more experienced contributors will review
-it, and suggest changes if needed.
+You are now ready to send a pull request to the pandas repository, to get your
+changes reviewed.
 
 First, push your changes to your GitHub fork:
 
     | ``git push -u origin <your-branch-name>``
 
-Then, visit https://github.com/python-sprints/pandas in your browser, and click
+Then, visit https://github.com/pandas-dev/pandas in your browser, and click
 on the "Compare & pull request" button in the yellow box above the repository
 files.
 
-Once your changes are reviewed and approved, you will receive an email on how
-to send the final pull request to pandas.
+6. Update your pull request based on feedback
+---------------------------------------------
```

Comment:
Added this section to explicitly explain how to update a PR, if that looks OK (sometimes people not familiar with this then open a new PR or so).